docs/docker.md

```diff
@@ -0,0 +1,72 @@
+# Running all the components with docker compose
+
+There are 4 docker compose files provided to run origin stack:
+
+-   based on the latest **stable** versions of NPM packages:
+    -   Vanilla Origin version with a **custom Device Registry** (`origin.release.yml`)
+    -   **I-REC compliant** Origin version (`origin.release.irec.yml`)
+-   based on the latest **canary** versions of NPM packages:
+    -   Vanilla Origin version with a **custom Device Registry** (`origin.local.yml`)
+    -   **I-REC compliant** Origin version (`origin.local.irec.yml`)
+
+All compose files require docker images to be built locally.
+
+## Building docker images locally
+
+<div class="admonition attention">
+  <p class="first admonition-title">Attention</p>
+  <p class="last">
+    Make sure you completed <a href="../getting-started">Getting started</a> in order to install and build Origin monorepo.
+  </p>
+</div>
+
+-   Check out the latest code from the [Origin repository](https://github.com/energywebfoundation/origin)
+-   Execute the following commands from the code folder to build the codebase:
+
+```shell
+rush update
+rush build #in case of errors it is necessary to execute rush rebuild
+```
+
+-   Build docker images
+
+    -   For the latest stable release:
+
+        ```shell
+        yarn build:containers:release
```

Comment:
This can also be removed from the README, as it will probably only be used by the CI/CD.

Reply:
do you mean building locally stable or canary images? It may be confusing if someone tries to run this setup without images built locally. Outdated versions will be fetched from the docker hub. If we do not plan to maintain them I would remove them from the hub.

Reply:
I meant just removing the instructions for stable from the README. Canary should be there, and that's the one that devs would be using for local development.

Devs shouldn't really run the stable releases in their environments, the CI/CD system should, so I'm not sure if it's useful having it in the README.